GameData/ROCapsules/PartConfigs/Vostok/VoskhodCapsule.cfg

```diff
@@ -1,195 +1,205 @@
-		StartActionName = Start CO2 Scrubber
-		StopActionName = Stop CO2 Scrubber
-		tag = Life Support
-		GeneratesHeat = False
-		UseSpecialistBonus = True
-		SpecialistEfficiencyFactor = 0.2
-		SpecialistBonusBase = 0.05
-		ExperienceEffect = ConverterSkill
-		EfficiencyBonus = 1
-		conversionRate = 3.0	// # of people - Figures based on per/person
-
-		INPUT_RESOURCE
+		name = ModuleFuelTanks
+		volume = 1
+		type = Fuselage
+		basemass = -1
+		TANK
 		{
-			ResourceName = CarbonDioxide
-			Ratio = 0.005921
-		}
+			name = ElectricCharge
+			amount = 1000
+			maxAmount = 1000
+		}	
+	}
 
-		INPUT_RESOURCE
-		{
-			ResourceName = ElectricCharge
-			Ratio = 0.01
-		}
+	MODULE
+	{
+		name = ModuleAnimateGeneric
+		animationName = Deploy
+		isOneShot = false
+		startEventGUIName = Extend Antenna
+		endEventGUIName = Retract Antenna
+		actionGUIName = Toggle Antenna
+		allowAnimationWhileShielded = False
+	}
 
-		INPUT_RESOURCE
-		{
-			ResourceName = PotassiumSuperoxide
-			Ratio = 0.00001735323
-		}
+	MODULE:NEEDS[ConnectedLivingSpace]
+	{
+		name = ModuleConnectedLivingSpace
+		passable = true
+		impassablenodes = bottom
+	}
 
-		OUTPUT_RESOURCE
-		{
-			ResourceName = Oxygen
-			Ratio = 0.006851
-			DumpExcess = True
-		}
+	MODULE:NEEDS[RasterPropMonitor]
+	{
+		name = RasterPropMonitorComputer
+	}
 
-		OUTPUT_RESOURCE
-		{
-			ResourceName = Waste
-			Ratio = 0.00006491
-			DumpExcess = False
-		}
+	//	================================================================================
+	//	Comms
+	//	================================================================================
+
+	MODULE:NEEDS[!RemoteTech]
+	{
+		name = ModuleDataTransmitter
+		antennaType = INTERNAL
+		packetInterval = 1.0
+		packetSize = 1.0
+		requiredResource = ElectricCharge
+		antennaPower = 400
+		packetResourceCost = 0.004
 	}
 	
-	MODULE
+	MODULE:NEEDS[RemoteTech]
 	{
-		name = ModuleConnectedLivingSpace
-		passable = true
-		impassablenodes = bottom
+		name = ModuleSPU
 	}
 	
+	MODULE:NEEDS[RemoteTech]
+	{
+		name = ModuleRTAntennaPassive
+		TechRequired = unlockParts
+
+		OmniRange = 1000000
+
+		TRANSMITTER
+		{
+			PacketInterval = 0.4
+			PacketSize = 0.27
+			PacketResourceCost = 0.025
+		}
+	}
+}
+
+//	================================================================================
+//	Configure RealAntennas
+//	================================================================================
+
+@PART[ROC-VoskhodCapsule]:BEFORE[RealAntennas]
+{
+	!MODULE[ModuleDataTransmitter],* {}
 	MODULE
 	{
-		name = ModuleFuelTanks
-		volume = 632.28
-		type = Fuselage
-		basemass = -1
+		name = ModuleRealAntenna
+		referenceGain = 1.5
+		RFBand = UHF		//Default to UHF-band
+	}
+}
+
+//	==================================================
+//	TAC Life Support compatibility.
+//	==================================================
+
+@PART[ROC-VoskhodCapsule]:FOR[RealismOverhaul]:NEEDS[TacLifeSupport]
+{
+	@MODULE[ModuleFuelTanks]
+	{
+		@volume += 631.28
+
```

Comment:
Rather than create volume only for TACLS (which is a Kerbalism compatibility problem), assign the appropriate volume in the base part's tankage.  Then let TACLS, or Kerbalism, or the player consume the volume instead.